배수연/4월 2주차/BOJ_G5_2589.java

```diff
@@ -0,0 +1,69 @@
+package boj;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.LinkedList;
+import java.util.Queue;
+import java.util.StringTokenizer;
+
+/**
+ * 보물섬
+ */
+
+public class BOJ_G5_2589 {
+	static int H, W;
+	static char[][] map;
+	static int[][] visited;
+	static int max;
+	static int[] dr = {-1, 1, 0, 0};
+	static int[] dc = {0, 0, -1, 1};
+
+	public static void main(String[] args) throws Exception {
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		StringTokenizer st = new StringTokenizer(br.readLine());
+		H = Integer.parseInt(st.nextToken());
+		W = Integer.parseInt(st.nextToken());
+		max = 0;
+		map = new char[H][W];
+		for(int h = 0 ; h<H ; h++) {
+			String line = br.readLine();
+			for(int w = 0 ; w<W ; w++) {
+				map[h][w] = line.charAt(w);
+			}
```

Comment:
`br.readLine` 을 `toCharArray()` 로 바로 받으셔도 좋을 것 같아요 !
```suggestion
			map[h] = br.readLine().toCharArray();
```